### ISSUE TITLE
cobbler image no longer adds menu entry (breaking memtest86+ in menu)

### ISSUE THREAD

**Issue by eikef** [cobbler/cobbler#3094]
### `cobbler image` is no longer correctly adding menu-entires to pxelinux.cfg

```
cobbler image add --name=memtest86plus --file=/boot/memtest86+-5.01 --image-type=direct
```

used to add an image of /boot/memtest86+-5.01 to the Cobbler PXE menu, which allowed you to electively boot into memtest86+. This worked as of Cobbler version 3.2.2.

In the current version this no longer works; the menu entry is not being generated

### Steps to reproduce

(0.) dnf install memtest86+
1. cobbler image add --name=memtest86plus --file=/boot/memtest86+-5.01 --image-type=direct
2. cobbler sync
3. `grep memtest86plus /var/lib/tftpboot/pxelinux.cfg/default` yields an empty result

### Expected behavior

3. `grep memtest86plus /var/lib/tftpboot/pxelinux.cfg/default` is not empty 

(on cobbler 3.2.2,

```
LABEL memtest86plus
        kernel /images2/memtest86plus
        MENU LABEL memtest86plus
        append  
        ipappend 2
```
used to be found there.)

### Cobbler version

````paste below
Cobbler 3.3.3
  source: b4df3716, Mon Apr 25 13:19:08 2022 +0200
  build time: Fri May  6 13:25:13 2022
````

### Operating system

AlmaLinux 8.5

### Additional information

There is a dirty workaround of adding Memtest86+ as a distro/profile with breed `windows` (since otherwise the kernel append lines make it crash otherwise; incidentally one will have to add an empty initrd that ends up being unused on windows breeds, but cobbler sync will fail otherwise):

```
mkdir /opt/memtest
cp /boot/memtest86+-5.01 /opt/memtest/vmlinuz
touch /opt/memtest/initrd
cobbler distro add --name=memtest86plus --breed windows --kernel=/opt/memtest/vmlinuz --initrd=/opt/memtest/initrd
cobbler profile add --name=memtest86plus --distro memtest86plus
```

**Comment by SchoolGuy** [comment]
Thanks for reporting this. Just one small question: Does that mean that the bug was introduced after Cobbler 3.2.2 according to your tests?

**Comment by eikef** [comment]
Yes (the 3.2.2 installation came from EPEL on CentOS8, while the 3.3.2 one is from a git checkout of the release33 branch on an AlmaLinux 8.5).
I haven't tried the EPEL 3.3.2 build thus far, though (though I wouldn't think there would be a difference).

**Comment by SchoolGuy** [comment]
This was fixed in #3205
